.gitmodules

```diff
@@ -29,7 +29,7 @@
 [submodule "MOM6"]
   path = MOM6-interface/MOM6
   url = https://github.com/NOAA-EMC/MOM6
-  branch = dev/emc
+  branch = dev/emc 
```

Comment:
Remove trailing whitespace.

Reply:
> Remove trailing whitespace.

@DusanJovic-NOAA thanks for the catch, done.